workflow-templates/im-deploy-windows-service.yml

```
@@ -0,0 +1,396 @@
+# Workflow Code: MaterialVolcano_v1    DO NOT REMOVE
+
+# Prerequisites
+# 1 - Make sure the Service account secrets have been added to the environment
+# 2 - Make sure the host windows server has been prepped to accept incoming WinRM Requests
+
+# TODO: Ensure each of the repo-level and env-level secrets used in this workflow have been populated by an admin in your repository.
+# TODO: Set up a deployment board if it does not already exist: https://github.com/im-practices/git-er-done/blob/main/actions/deployment-board.md
+
+name: Deploy On Prem Windows Service
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: The repository tag pointing to the code that should be deployed.  For Production, only tags reachable by the default branch will be accepted.
+        required: true
+      environment:
+        description: The environment to deploy to.
+        required: true
+env:
+  GITHUB_REF: ${{ github.event.inputs.tag }}
+  SERVICE_EXE_NAME: '' # TDOO: Set the name of the service exe - To use this value: ${{ needs.set-vars.outputs.SERVICE_PATH }}
```

This is an environment variable so we don't need the rest of the comment about how to use it.
```suggestion
  SERVICE_EXE_NAME: '' # TODO: Set the name of the service exe 
```